.eslintrc.json

```diff
@@ -0,0 +1,46 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6": true,
+        "node": true,
+        "jquery": true,
+        "jest": true
+    },
+    "globals": {
+        "app": "readonly",
+        "isDev": "readonly",
+        "Sentry": "readonly",
+        "loadScript": "readonly",
+        "verseListTemplate": "readonly",
+        "ipcNsi": "readonly",
+        "ipcSettings": "readonly",
+        "ipcDb": "readonly",
+        "ipcGeneral": "readonly",
+        "app_controller": "readonly",
+        "tags_controller": "readonly",
+        "theme_controller": "readonly",
+        "uiHelper": "readonly",
+        "i18n": "readonly",
+        "platformHelper": "readonly"
+    },
+    "extends": "eslint:recommended",
```

Comment:
@tobias-klein 
Please, note that this PR will introduce eslint rules a bit extended rules from #320 
Rules are based on ESLint recommendations

Reply:
Ok, cool :)